src/contacts/ApplicationContacts.java

```diff
@@ -0,0 +1,80 @@
+            IEditRecordActionProcessor editSurnamePersonProcessor = new EditSurnamePersonProcessor(recordService);
+            IEditRecordActionProcessor editPhoneNumberRecordProcessor = new EditPhoneNumberRecordProcessor(recordService);
+            IEditRecordActionProcessor editGenderPersonProcessor = new EditGenderPersonProcessor(recordService);
+
+            IEditRecordProcessorFactory editPersonProcessorFactory = new EditPersonProcessorFactory(
+                    List.of(editNamePersonProcessor, editSurnamePersonProcessor,
+                            editPhoneNumberRecordProcessor, editGenderPersonProcessor, editAddressOrganizationProcessor));
+
+            IActionProcessor chooseTypeRecordProcessor = new ChooseTypeRecordProcessor(recordService, recordProcessorFactory);
+
+            IActionProcessor countRecordProcessor = new CountRecordProcessor(recordService);
+
+            IActionProcessor searchPhoneNumberRecordProcessor = new SearchPhoneNumberRecordProcessor(recordService);
+            IActionProcessor searchBackRecordProcessor = new SearchBackRecordProcessor();
+            IActionProcessor searchAgainRecordProcessor = new SearchAgainRecordProcessor(recordService);
+            ISearchRecordProcessorFactory searchRecordProcessorFactory = new SearchRecordProcessorFactory(
+                    List.of(searchPhoneNumberRecordProcessor, searchBackRecordProcessor, searchAgainRecordProcessor));
+
+            IActionProcessor editRecordProcessor = new EditRecordProcessor(recordService,
+                    editPersonProcessorFactory);
+            IActionProcessor removeRecordProcessor = new RemoveRecordProcessor(recordService);
+            IActionProcessor exitApplicationProcessor = new ExitAppActionProcessor();
+
+            IActionProcessor menuApplicationProcessor = new MenuApplicationProcessor();
+
+            IProcessorFactory processorFactory = new ProcessorFactory(List.of(countRecordProcessor,
+                    editRecordProcessor, removeRecordProcessor, exitApplicationProcessor, chooseTypeRecordProcessor, menuApplicationProcessor));
+
+            IActionProcessor searchRecordProcessor = new SearchRecordProcessor(recordService, processorFactory);
+
+            IActionProcessor listRecordProcessor = new ListRecordProcessor(recordService);
+            processorFactory = new ProcessorFactory(List.of(countRecordProcessor, editRecordProcessor,
+                    removeRecordProcessor, exitApplicationProcessor, chooseTypeRecordProcessor, listRecordProcessor,
+                    menuApplicationProcessor, searchRecordProcessor));
+
+            PhoneBook phoneBook = new PhoneBook(processorFactory);
+            phoneBook.runPhoneBook();
+        } catch (NumberFormatException | IndexOutOfBoundsException exception) {
+            System.out.println(exception.getMessage());
+        } catch (IOException exception) {
+            throw new RuntimeException(exception);
+        } catch (ClassNotFoundException e) {
```

Comment:
why do you need this?

Reply:
Done, deleted

Reply:
When I use classes for writing or reading from File, they can throws ClassNotFoundException